app/controllers/api_frontend/v1/projects_controller.rb

```diff
@@ -0,0 +1,56 @@
+module APIFrontend
+  module V1
+    class ProjectsController < ApiController
+      before_action :set_project, only: [:show, :update, :destroy]
+
+      # GET /projects
+      def index
+        respond_with policy_scope(Project), each_serializer: ProjectSerializer
+      end
+
+      # GET /projects/1
+      def show
+        respond_with @project, serializer: ProjectSerializer
+      end
+
+      # POST /projects
+      def create
+        @project = current_user.projects.build(project_params)
+
+        if @project.save
+          respond_with @project, serializer: ProjectSerializer, status: 201
+        else
+          render status: 400, json: { errors: @project.errors }
+        end
+      end
+
+      # PATCH/PUT /projects/1
+      def update
+        if @project.update(project_params)
+          render status: 200, json: { project: @project }
+          # respond_with @project, serializer: ProjectSerializer, status: 200
```

Comment:
@ondrejbartas it looks better with `respond_with` but `respond_with` is deprecated in current version of rails & it doesn't work for update, it responds with empty string and status 204 - no content and I don't know why.

Could you take a look at it if we want to keep `respond_with` here?